src/com/mie/dao/PostDao.java

```diff
@@ -0,0 +1,188 @@
+package com.mie.dao;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+
+import com.mie.model.Post;
+import com.mie.model.User;
+import com.mie.util.DbUtil;
+import com.sun.jmx.snmp.Timestamp;
```

Comment:
Think this is the wrong import, should be import [java.sql.Timestamp](https://docs.oracle.com/cd/E13222_01/wls/docs45/classdocs/java.sql.Timestamp.html)